pkg/controller/user-cluster-controller-manager/flatcar/flatcar_controller.go

```diff
@@ -39,7 +39,9 @@ import (
 )
 
 const (
-	ControllerName = "kubermatic_flatcar_controller"
+	// This controller is responsible for ensuring that the flatcar-linux-update-operator is installed when we have a healthy(running) flatcar
+	// node in our cluster.
+	ControllerName = "kkp-flatcar-update-operator-manager"
```

Comment:
Should IMHO drop the `manager` suffix and be something like `kkp-flatcar-operator-controller`.

Reply:
I TBH think `-manager` makes so much sense here. Because this controller will be responsible for managing the lifecycle of the `flatcar-update-operator`. I know it might convey the notion that manager = controller-manager. But it's not the case here.

Reply:
@xrstf any thoughts? Maybe I'm pushing back too much against the `-manager` suffix, but I really dislike it to be honest.

Reply:
To me a `-manager` is something that contains multiple controllers, so I would also argue that we should not put `-manager` as the suffix for a single controller.

Reply:
Then we have a decision ✌️